src/content/docs/reference/api/profile-git-credentials.md

```
@@ -0,0 +1,100 @@
+
+```
+org:{profile-name}
+```
+
+Example: `POST /organization/git-credentials/org:deploy`
+
+### Request body
+
+The request body follows Git's credential helper input format:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)